File: Python/ex10.py
class Calculadora:
    def somar_dois(self, a, b):
        return a + b

    def somar_tres(self, a, b, c):
        return a + b + c


def main():
    calc = Calculadora()

    print("Soma de 5 e 10:", calc.somar_dois(5, 10))
    print("Soma de 5, 10 e 15:", calc.somar_tres(5, 10, 15))


if __name__ == "__main__":
    main()

File: Golang/ex10.go
package main

import "fmt"

func somarDois(a int, b int) int {
	return a + b
}

func somarTres(a int, b int, c int) int {
	return a + b + c
}

func main() {
	fmt.Println("Soma de 5 e 10:", somarDois(5, 10))
	fmt.Println("Soma de 5, 10 e 15:", somarTres(5, 10, 15))
}
